documentation/library_guide/kernel_templates/single_pass_scan.rst

```diff
@@ -0,0 +1,208 @@
+|                                               | - :ref:`oneapi::dpl::experimental::ranges::views::all               |
+|                                               |   <viewable-ranges>` (2),                                           |
+|                                               | - :ref:`oneapi::dpl::experimental::ranges::views::subrange          |
+|                                               |   <viewable-ranges>` (2),                                           |
+|                                               | - :ref:`USM pointers <use-usm>` (1),                                |
+|                                               | - :ref:`oneapi::dpl::begin and oneapi::dpl::end                     |
+|                                               |   <use-buffer-wrappers>` (1).                                       |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+|  ``binary_op``                                | A function object that will applied to the elements of the input    |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+|  ``param``                                    | A :doc:`kernel_param <../kernel_configuration>` object.             |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+
+
+**Type Requirements**:
+
+- The element type of sequence to scan must be a C++ integral or floating-point type
+  other than ``bool`` with a width of up to 64 bits.
+- The result is non-deterministic if the binary operator is non-associative (such as in floating-point addition).
+
+
+**Current limitations**:
+
+- The function will internally block until the issued kernels have completed execution.
+  Although intended in the future to be an asynchronous call, the algorithm is currently synchronous.
+- The SYCL device associated with the provided queue must support 64-bit atomic operations
+- The binary operator must have a known identity. That is, ``sycl::has_known_identity<Op>::value`` must
+  evaluate to true. Such operators are listed in the `SYCL 2020 specification <https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.identities>`_.
+
+Return Value
+------------
+
+A ``sycl::event`` object representing the status of the algorithm execution.
+
+--------------
+Usage Examples
+--------------
+
+
+``inclusive_scan`` Example
+----------------------
```

Comment:
We don't use monospace font in headers, please remove the ```` from inclusive_scan. The header underline needs to go the full length of the header, otherwise it renders incorrectly. Please fix the length.